apps/frontend/src/libs/components/sidebar/styles.module.css

```diff
@@ -0,0 +1,16 @@
+.sidebarContainer {
+	background-color: var(--background-gray);
+}
+
+.logoContainer {
+	padding-top: 24px;
```

Comment:
You can write padding: 24px 0 25px 30px

Reply:
it passes the lint stage, but linter has rules for no 0 values. While it passes when 0 provided without px, but I feel like it beats the reason for which this rule is present